### PR TITLE
make PAGE_SIZE configurable for Slack API list calls

### PR DESCRIPTION
**Pull request by kintarowins**
###  Summary

Describe the goal of this PR. Mention any related Issue numbers.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/hubot-slack/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: src/client.coffee [hunk 1]
```diff
@@ -43,6 +44,9 @@ class SlackClient
     @rtm = new RtmClient options.token, options.rtm
     @web = new WebClient options.token, { maxRequestConcurrency: 1 }
 
+    unless isNaN(options.apiPageSize)
+      SlackClient.PAGE_SIZE = parseInt(options.apiPageSize, 10)
```
Comment (seratch) [inline src/client.coffee:48]: I know this is a pretty easy to make this work. However, now that the value is no longer a constant, we may want to use an instance filed like `@page_size` over a class field plus replace the references to `SlackClient.PAGE_SIZE` in the code.

Comment (kintarowins) [reply src/client.coffee]: Thanks, yeah I was thinking that too, needed someone to confirm my own doubt. I used @apiPageSize to follow existing camelCase naming convention.

**Comment by seratch** [comment]
@kintarowins Thanks a lot for promptly updating the PR. Right now, I'm a bit busy for other tasks but I will be checking if this change works as we expect. 

By the way, can you sign our CLA (see the above bot message for details)? 🙇 Without it, we are unable to merge your change.

**Comment by seratch** [comment]
@kintarowins Thanks for signing it. Regarding the page size, I am reluctant to change the default value for keeping backward compatibility. Your change enables developers to customize the size. Let's keep the default as-is.

I did not have time to do some tests today but I'll check this sometime this week.

**Comment by seratch** [comment]
I just verified if your change works with my Hubot example apps. Before merging this pull request, I will migrate the CI builds to GitHub Actions.

Thanks for your contribution!

**Review by seratch — APPROVED** [review]

**Pull request merged**